### PR TITLE
Adding superhero video

### PR DESCRIPTION
**Pull request by iamdeaneyelid**
@jonsherrard @ecoad

### CODE REVIEW

File: app/CustomFields/HeaderDefinition.php [hunk 1]
```diff
@@ -37,6 +38,44 @@ public static function get($post_type) {
           ),
         ),
         array (
+          'key' => $key . '_video_file',
+          'label' => 'Background video',
+          'name' => 'background_video',
```
Comment (ecoad) [inline app/CustomFields/HeaderDefinition.php:43]: This `name` should match `key`

`'key' => $key . '_video_file'`
`'name' => 'video_file',`

Or rename them both to `background_video`

File: app/CustomFields/HeaderDefinition.php [hunk 2]
```diff
@@ -37,6 +38,44 @@ public static function get($post_type) {
...
+              array (
+                'field' => $key . '_type',
+                'operator' => '==',
+                'value' => 'super-hero-video',
+              ),
+            ),
+          ),
+        ),
+        array (
+          'key' => $key .'video_poster',
```
Comment (ecoad) [inline app/CustomFields/HeaderDefinition.php]: Missing an underscore in here `'_video_poster'`

File: app/CustomFields/HeaderDefinition.php [hunk 3]
```diff
@@ -37,6 +38,44 @@ public static function get($post_type) {
...
+          'label' => 'Poster',
+          'name' => 'poster',
```
Comment (ecoad) [inline app/CustomFields/HeaderDefinition.php]: Again, should match `key`

File: app/CustomFields/HeaderDefinition.php [hunk 4]
```diff
@@ -83,15 +122,21 @@ public static function get($post_type) {
             'parallax' => 'Enable parallax effect',
             'carousel-buttons' => 'Enable previous/next carousel buttons (if used)',
             'scroll-down-button' => 'Enable scroll down button (if full-height is used)',
-            'display-post-category' => 'Display post category'
+            'display-post-category' => 'Display post category',
+            'autoplay' => 'Enable autoplay when the video is in view',
```
Comment (ecoad) [inline app/CustomFields/HeaderDefinition.php]: Autoplay, loop, mute etc are all video hero specific, we should split these options up in to a separate ACF field which conditionally shows when video hero is selected

File: app/CustomFields/HeaderDefinition.php [hunk 5]
```diff
@@ -83,15 +122,21 @@ public static function get($post_type) {
...
+            'mute' => 'Mute the video',
           ),
           'default_value' => array (
             'full-height' => '',
             'parallax' => '',
             'carousel-buttons' => 'carousel-buttons',
             'scroll-down-button' => 'scroll-down-button',
-            'display-post-category' => 'display-post-category'
-            ),
+            'display-post-category' => 'display-post-category',
```
Comment (ecoad) [inline app/CustomFields/HeaderDefinition.php]: Remember to update these too.

**Comment by jonsherrard** [comment]
👍

**Pull request merged**